SourceEditor/dependencies.gradle

```diff
@@ -16,10 +16,9 @@ ext {
     duo_url = "https://pkgs.dev.azure.com/MicrosoftDeviceSDK/DuoSDK-Public/_packaging/Duo-SDK-Feed/maven/v1"
 
     config = [
+            duoSdkVersion            : "1.0.0-beta1",
```

Comment:
I believe there are already other PRs that handle this, so you will probably have conflicts with this ;-)

Reply:
Actually no, because each app has his own dependencies file.

Reply:
wops!! true that! I thought it was like in the kotlin samples. So many PRs today! 😅🤣